src/current/cockroachcloud/network-authorization.md

```diff
@@ -41,10 +48,20 @@ Authorized network access can be managed from the {{ site.data.products.db }} co
 
 `https://cockroachlabs.cloud/cluster/{ your cluster UUID}/networking`
 
-You can add up to 20 IP addresses to your allowlist. If your application servers’ IP addresses are not static, or you want to limit your cluster's exposure to the public network, you can use VPC Peering or AWS PrivateLink instead (Dedicated clusters only).
+Serverless and Dedicated clusters support different maximum number of IP allowlist rules/authorized sub-networks:
 
-Refer to:
+Cluster Type | IP allowlist rule max
+--------|------------
+Dedicated|20
+Serverless|50
 
+{{site.data.alerts.callout_info}}
+While developing and testing your application, you may add `0.0.0.0/0` to the allowlist, which allows all networks. However, before moving into production, make sure you delete the `0.0.0.0/0` network.
+{{site.data.alerts.end}}
+
+If your application servers’ IP addresses are not static, or you want to limit your cluster's exposure to the public network, you can connect to your {{ site.data.products.dedicated }} clusters using VPC Peering or AWS PrivateLink instead.
+
+Refer to:
```

Comment:
These points are not rendering correctly. I think you may need to leave a line space in the markdown after "Refer to:"

Reply:
thanks